views/js/pciCreator/ims/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -306,17 +316,46 @@ define([
                 this.initProgressBar();
                 this.initMeter();
                 this.initControls();
+                this.initBeepPlayer();
                 this.updateResetCount();
                 this.initRecording();
+
+                if (this.config.enableDomEvents) {
+                    // incoming events
+
+                    this.$container.get(0).addEventListener('config-change', ({ detail: newConfig }) => {
+                        console.log('PCI received config-change', newConfig);
+                        this.render(newConfig);
+                    });
+
+                    // outgoing events
+
+                    this.recorder.on('stop', () => {
+                        console.log('PCI dispatch recorder-stop');
+                        this.$container.get(0).dispatchEvent(
+                            new CustomEvent('recorder-stop', {
+                                recordsAttempts: this._recordsAttempts
+                            })
+                        );
+                    });
+
+                    this.player.on('playbackend', () => {
+                        console.log('PCI dispatch playback-end');
```

Comment:
Don't forget to remove  `console.log`s for these events

Reply:
Done in https://github.com/oat-sa/extension-tao-itemqti-pci/pull/367/commits/9e8893bd0ed456220986ea256119bfc7a32269b5